ocs_ci/helpers/dr_helpers.py

```diff
@@ -1083,3 +1088,129 @@ def verify_drpolicy_cli(switch_ctx=None):
+        if (name != "all-openshift-clusters") and (name != "global"):
+            namespace = placement["metadata"]["namespace"]
+            path = "/spec/predicates/0/requiredClusterSelector/labelSelector/matchExpressions/0/values/0"
+            params = f"""[{{"op": "replace", "path": "{path}", "value": "{secondary_cluster_name}"}}]"""
+            cmd = f"oc patch placement {name} -n {namespace}  -p '{params}' --type=json"
+            run_cmd(cmd)
+
+    # Delete all drpc
+    run_cmd("oc delete drpc --all -A")
+    sample = TimeoutSampler(
+        timeout=300,
+        sleep=5,
+        func=run_cmd_verify_cli_output,
+        cmd="oc get drpc -A",
+        expected_output_lst="No resources found",
+    )
+    if not sample.wait_for_func_status(result=True):
+        raise Exception("All drpcs are not deleted")
+
+    # Remove annotation from placements
+    for placement in placements:
+        name = placement["metadata"]["name"]
+        if (name != "all-openshift-clusters") and (name != "global"):
+            namespace = placement["metadata"]["namespace"]
+            path = "/metadata/annotations/cluster.open-cluster-management.io~1experimental-scheduling-disable"
+            params = f"""[{{"op": "remove", "path": "{path}"}}]"""
+            cmd = f"oc patch {constants.PLACEMENT} {name} -n {namespace} -p '{params}' --type=json"
+            run_cmd(cmd)
+
+    config.switch_ctx(old_ctx)
+    return placement_obj
+
+
+def apply_drpolicy_to_workload(workload, drcluster_name):
+    """
+    Function for applying drpolicy to indiviusual workload
+
+    Args:
+    workload(List): List of workload objects
+    drcluster_name(str): Name of the DRcluster on which workloads belongs
+    """
+    for wl in workload:
+        drpc_yaml_data = templating.load_yaml(wl.drpc_yaml_file)
+        drpc_yaml_data["spec"]["preferredCluster"] = drcluster_name
+        templating.dump_data_to_temp_yaml(drpc_yaml_data, wl.drpc_yaml_file)
+
+        config.switch_acm_ctx()
+        run_cmd(f"oc create -k {wl.drpc_yaml_file}")
+
+
+def replace_cluster(workload, primary_cluster_name, secondary_cluster_name):
+
+    """
+    Function to do core replace cluster task
+
+    Args:
+    workload(List): List of workload objects
+    primary_cluster_name (str): Name of the primary DRcluster
+    secondary_cluster_name(str): Name of the secondary DRcluster
+    """
+
+    # Delete dr cluster
+    config.switch_acm_ctx()
+    run_cmd(cmd=f"oc delete drcluster {primary_cluster_name} --wait=false")
+
+    # Disable DR on hub for each app
+    placement_obj = disable_dr_from_app(secondary_cluster_name)
+    logger.info("DR configuration is successfully disabled on each app")
+
+    # Remove DR configuration from hub and surviving cluster
+    logger.info("Running Remove DR configuration script..")
+    run_cmd(cmd=f"chmod +x {constants.REMOVE_DR_EACH_MANAGED_CLUSTER}")
+    run_cmd(cmd=f"sh {constants.REMOVE_DR_EACH_MANAGED_CLUSTER}")
+
```

Comment:
@avd-sagare Can we have sleep time before adding label to openshift-operators, as it takes some time for the namespace to be created after the running the script.

Reply:
Added